smbclient/_os.py

```diff
@@ -597,6 +609,11 @@ def stat(path, follow_symlinks=True, **kwargs):
         st_chgtime_ns=chgtime_ns,
         st_file_attributes=file_attributes.get_value(),
         st_reparse_tag=reparse_tag,
+        st_full_size_total_allocation_units=full_size['total_allocation_units'].get_value(),
```

Comment:
I don't think having this as part of stat is a good idea, this information is info on the file system and not the file specified. While we do get the FsVolumeInformation that is purely because Python's `stat` returns the the volume ID for a file so it can compare whether 2 files are the same.

The trouble is I don't know how we want to expose this information. Do you have any ideas how you think it should look?

Reply:
Maybe I will create new function `get_full_size_information` that will return this information?

Reply:
Let's just do that for now and get your contribution in, I don't want to hold this up because I haven't decided how to expose it yet. If I decide to move it around in another PR before the next release I'll be sure to let you know in the PR that does that.

Reply:
Ok, Thanks